docs/docs/troubleshooting.md

```diff
@@ -26,28 +26,35 @@ Variations of the warnings shown below occur when flashing the `<firmware>.uf2`
 An error along the lines of `CMake Error at (zmk directory)/zephyr/cmake/generic_toolchain.cmake:64 (include): include could not find load file:` during firmware compilation indicates that the Zephyr Environment Variables are not properly defined.
 For more information, click [here](../docs/development/setup.md#environment-variables).
 
-### dtlib.DTError
+### West build errors
```

Comment:
The other headings use title-case, whereas this one currently doesn't:

<img width="250" alt="image" src="https://user-images.githubusercontent.com/60922914/161935459-11cd74a3-1041-4fb5-b7a2-a52af9453704.png">


```suggestion
### West Build Errors
```

Reply:
@dxmh Don't apologise, and thank you for the meticulous review!